husky_components/datagrid/main.js

```diff
@@ -572,11 +577,8 @@
                     direction: null
                 };
 
-                // Should only be be called once
-                this.bindCustomEvents();
-
-                this.initRender();
-
+                this.bindCustomEvents(); // Should only be be called once
+                this.initializeDecoratorsAndRender();
```

Comment:
@nnatter can we split this up in a 'initializeDecorators' and a 'render' method?